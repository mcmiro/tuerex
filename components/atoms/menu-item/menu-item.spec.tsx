/* eslint-disable no-use-before-define */
import * as React from 'react';
import { screen, render } from '@testing-library/react';
import MenuItem from './';

describe('Test MenuItem component', () => {
  it('Should pass when item and hover function and placeholder is additional', () => {
    const mockItem = {
      name: '',
      url: '',
    };
    const hoverMock = jest.fn();

    render(<MenuItem data-testid="test" item={mockItem} onHover={hoverMock} />);

    const renderFiledComponent = screen.queryAllByTestId('test');
    expect(renderFiledComponent).not.toBeNull();
    expect(renderFiledComponent).toBeDefined();
    expect(renderFiledComponent).toMatchSnapshot();
  });
});
