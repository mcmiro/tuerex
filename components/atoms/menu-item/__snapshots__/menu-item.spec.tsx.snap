// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MenuItem component Should pass when item and hover function and placeholder is additional 1`] = `[]`;
